Makefile

```diff
@@ -82,10 +82,6 @@ dependencies: ## Install dependencies
 	mix deps.get
 	npm install --prefix assets
 
-.PHONY: clean
-clean: ## Clean dependencies
```

Comment:
I would personally keep `clean`, it is usefull when `check-unused-dependencies` raises…

Reply:
But it’s not documented nor used anywhere in the projet.

`make clean` is often used in `Makefile`s to remove temporary files or artifacts that were generated during the build or install, so I think the name is misleading.

I guess we could add `make dependencies-clean` to be more explicit but I don’t think it’s right to just add an abstract target to "hide" the `mix deps.clean --unused --unlock` command, especially when the command can be shown to the user by Mix in error/warning messages.

What do you think?

Reply:
If `make check-unused-dependencies` explicitly prints the clean command I think this is perfect!